### PR TITLE
Node postgres

### PR DESCRIPTION
**Pull request by jc-frc725**
Back end:
- Removed mongoose/mongoDB drivers
- replaced with node-postgres

Front end:
- slight changes to top level components to make sure state matches response data
- slight calendar controller changes to make sure it can handle data correctly

### CODE REVIEW

**Review by apat2010 — APPROVED** [review]
Reviewed and all looks good. Was surprised by all of the different pg dependencies for dates/intervals. Great/clean code!

File: server/controller/properties.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const db = require('../database');
+
+const properties = (request, response) => {
+  const { propertyId } = request.params;
+  db.query(`SELECT * FROM properties WHERE propertyId = ${propertyId}`)
+    .then(data => {
+      response.send(data.rows[0]);
+    });
+}
```
Comment (christina003) [inline server/controller/properties.js:9]: Are we supposed to be sending http codes back here with the data?

**Review by christina003 — COMMENTED** [review]
Changes look good!

File: server/controller/properties.js (hunk 1 above)
Comment (jc-frc725) [reply server/controller/properties.js:9]: Will fix this

**Pull request closed without merging**